libvast/src/taxonomies.cpp

```diff
@@ -25,6 +26,46 @@
 
 namespace vast {
 
+static caf::error to_concepts_impl(const data& d, concepts_type& out) {
```

Comment:
If we keep the error handling as is, this function should be called `extract_concepts` instead. I'm not sure which way we'll want to go with this yet.

Reply:
Yeah, I like your proposal better.